rotkehlchen/tests/api/test_trades.py

```diff
@@ -436,6 +436,57 @@ def test_add_trades(rotkehlchen_api_server):
         status_code=HTTPStatus.BAD_REQUEST,
     )
 
+    # Test with fee & without fee_currency
+    fee_and_no_fee_currency_trade = {
+        'timestamp': 1595640208,
+        'location': 'external',
+        'base_asset': 'ETH',
+        'quote_asset': 'USD',
+        'trade_type': 'buy',
+        'amount': '1.5541',
+        'rate': '22.1',
+        'fee': '0.55',
+    }
+
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            "tradesresource",
+        ), json=fee_and_no_fee_currency_trade,
+    )
+
```

Comment:
nitpick: don't add unneeded lines. Part of the test that should be bundled together should have no newlines
```suggestion
```